webapp/portlet/src/main/webapp/vue-apps/spaces-navigation/components/ExoSpacesHamburgerNavigation.vue

```diff
@@ -24,7 +24,12 @@
         <v-list-item-content class="subtitle-1 titleLabel">
           {{ $t('menu.spaces.lastVisitedSpaces') }}
         </v-list-item-content>
-        <v-list-item-action v-if="toggleArrow" class="my-0">
+        <v-list-item-action v-if="toggleArrow" class="my-0 d-flex flex-row align-center">
+          <v-btn icon @click="leftNavigationActionEvent('addNewSpace')">
```

Comment:
@SaraBoutej  to change by:
`
          <v-btn icon :href="allSpacesLink">
`
it's better to never user a `button` to make the job of a `link`